$NetBSD: patch-ab,v 1.4 2007/09/25 04:18:26 bjs Exp $

We really don't want to use getproto reentrant functions.  Also,
add missing extern decls.

--- pr/include/md/_netbsd.h.orig	2006-04-05 17:38:13.000000000 -0400
+++ pr/include/md/_netbsd.h
@@ -47,6 +47,8 @@
 #define _PR_SI_ARCHITECTURE "x86"
 #elif defined(__alpha__)
 #define _PR_SI_ARCHITECTURE "alpha"
+#elif defined(__amd64__)
+#define _PR_SI_ARCHITECTURE "amd64"
 #elif defined(__m68k__)
 #define _PR_SI_ARCHITECTURE "m68k"
 #elif defined(__powerpc__)
@@ -92,12 +94,6 @@
 #define _PR_INET6_PROBE
 #endif
 
-#if __NetBSD_Version__ >= 106370000
-/* NetBSD 1.6ZK */
-#define _PR_HAVE_GETPROTO_R
-#define _PR_HAVE_GETPROTO_R_INT
-#endif
-
 #define USE_SETJMP
 
 #ifndef _PR_PTHREADS
@@ -243,6 +239,10 @@ struct _MDCPU {
 
 #endif /* ! _PR_PTHREADS */
 
+extern void _MD_EarlyInit(void);
+extern PRIntervalTime _PR_UNIX_GetInterval(void);
+extern PRIntervalTime _PR_UNIX_TicksPerSecond(void);
+
 #define _MD_EARLY_INIT                  _MD_EarlyInit
 #define _MD_FINAL_INIT			_PR_UnixInit
 #define _MD_GET_INTERVAL                  _PR_UNIX_GetInterval
